src/main/java/com/google/alpollo/AnalysisHelper.java

```diff
@@ -0,0 +1,29 @@
+package com.google.alpollo;
+
+import java.io.IOException;
+import com.google.api.gax.rpc.FixedHeaderProvider;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.LanguageServiceSettings;
+import com.google.cloud.language.v1.Sentiment;
+
+/**
+ * Helper class with methods that use the Natural Language API or use the data that comes
+ * from the API.
+ */
+public class AnalysisHelper {
```

Comment:
If this class is just for static helper methods, create a private constructor for it so that it can never be instantiated by mistake

Reply:
Already ahead of you ;)